pkg/cmd/create/root.go

```diff
@@ -122,9 +122,17 @@ func create(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	subDomain := "argocd."
+	subPath := ""
+
+	if pathRouting == true {
+		subDomain = ""
+		subPath = "argocd"
+	}
+
 	fmt.Print("\n\n########################### Finished Creating IDP Successfully! ############################\n\n\n")
-	fmt.Printf("Can Access ArgoCD at https://argocd.cnoe.localtest.me:%s/\nUsername: admin\n", port)
-	fmt.Print(`Password can be retrieved by running: kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d`, "\n")
+	fmt.Printf("Can Access ArgoCD at %s\nUsername: admin\n", fmt.Sprintf("https://%s%s:%s/%s", subDomain, host, port, subPath))
```

Comment:
Protocol is also supported. We may as well put it there. 

```suggestion
	fmt.Printf("Can Access ArgoCD at %s\nUsername: admin\n", fmt.Sprintf("%s://%s%s:%s/%s", protocol, subDomain, host, port, subPath))
```